src/testScripts/sdkV2-scripts.ts

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as dotenv from 'dotenv';
+
+import { ethers } from 'ethers';
+import { CegaEvmSDKV2, EthereumAlchemyGasStation, types, GasStation } from '..';
+
+dotenv.config();
+
+const config = {
+  ethereum: {
+    RPC_URL: process.env.ETH_RPC_URL,
+    cegaEntryAddress: '0x0730AA138062D8Cc54510aa939b533ba7c30f26B' as types.EvmAddress,
```

Comment:
We deployed on ethereum already?

Reply:
no. I missed this when I copied the config object, and tried to change the variables.

will remove